Services/JobRunner/Jobs/DataXDeployJob.cs

```diff
@@ -56,6 +56,7 @@ public async Task RunAsync()
             {
                 context[Context.ServiceUrl] = _config.ServiceUrl;
                 context[Context.ApplicationId] = KeyVault.GetSecretFromKeyvault(_config.ApplicationId);
+                context[Context.TrustSsl] = _config.TrustSsl;

```

Comment:
comments, what is this new param for?

Reply:
Comments added to 
Services/JobRunner/AppConfig.cs

Tests/DataXScenarios/DataXScenarios/Context.cs